testsuite/features/support/xmlrpc_client.rb

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
+require 'xmlrpc/client'
+
+# Wrapper class for XML-RPC client library
+class XmlrpcClient
```

Comment:
Matter of taste, but I prefer XMLRPCClient :P

Reply:
I prefer to camel-case with one caps per "word". Matter of taste indeed.